subgit/importer/git_importer.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+
+# python std lib
+import json
+import logging
+import os
+import subprocess
+
+# subgit imports
+from subgit.core import SubGit
+
+# 3rd party imports
+from ruamel import yaml
+
+
+log = logging.getLogger(__name__)
+
+
+class GitImport(SubGit):
+    def __init__(self, config_file_name=None, answer_yes=None, is_archived=False):
+        self.answer_yes = answer_yes
+        self.is_archived = is_archived
+
+        # Defaults config file to '.subgit-github.yml' if nothing is specified
+        if not config_file_name:
+            self.subgit_config_file_name = ".subgit.yml"
+
+            if self.is_archived:
+                self.subgit_config_file_name = ".subgit-archived.yml"
+
+            self.subgit_config_file_path = os.path.join(os.getcwd(), self.subgit_config_file_name)
+        else:
+            self.subgit_config_file_name = config_file_name
+            self.subgit_config_file_path = os.path.join(os.getcwd(), self.subgit_config_file_name)
+
+    def _if_github_cli(self, c):
+        if c == "github":
+            shell_command = "gh"
+        else:
+            shell_command = "gitlab"
+
+        try:
+            out = subprocess.run([
+                    shell_command,
+                    "--help"
+                ],
+                shell=False,
+                capture_output=True,
+            )
+        except FileNotFoundError:
+            return 1
+
+        return out.returncode
+
+    def import_github(self, owner):
+        """
+        Given a username or organisation name, this method lists all repos connected to it
+        on github and writes a subgit config file.
+        """
+        if self._if_github_cli("github") != 0:
+            log.error("Github cli not installed. Exiting subgit...")
+            return 1
+
+        out = subprocess.run([
+                "gh",
+                "repo",
+                "list",
+                f"{owner}",
+                "--json",
+                "id,name,defaultBranchRef,sshUrl,isArchived",
+                "-L",
+                "100"
+            ],
```

Comment:
```suggestion
        out = subprocess.run([
                "gh", "repo", "list",
                f"{owner}",
                "--json", "id,name,defaultBranchRef,sshUrl,isArchived",
                "-L", "100"
            ],
```

So this suggestion is very unconventional as it would put multiple strings in a list on the same line, but at the same time it would increase the readability of the code as we join together what arguments really belong to eachother much more clearly.

You can choose here to keep as is or take this suggestion. by taking it we break pep8 a bit tho :P

Reply:
Given that it's structured in a similar way in the other method with multiple strings in the same row, I'd say you have a good point :P